conda.recipe/build.sh

```diff
@@ -0,0 +1,43 @@
+if [ "$(uname)" == "Darwin" ]; then
+  # C++11 finagling for Mac OSX
+  export CC=clang
+  export CXX=clang++
+  export MACOSX_VERSION_MIN="10.9"
+  CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  CXXFLAGS="${CXXFLAGS} -Wno-error=unused-command-line-argument"
+  export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export LINKFLAGS="${LDFLAGS}"
+  export MACOSX_DEPLOYMENT_TARGET=10.9
+
+  # make sure clang-format is installed
+  # e.g. brew install clang-format
+
+  # make sure autoreconf can be found
+  # e.g. brew install autoconf
+
+  # for graphviz, also brew install autogen libtool
+fi
+
+make install PREFIX=$PREFIX
+
+# add the /util directory to the PATH inside this conda env
```

Comment:
Not sure what a conda env is, but why do we want to expose everything in util/?  Most of them aren't standalone executables, i.e., won't work outside an env set up by bin/deepdive.. so this will introduce a lot of broken commands to the conda env.